### DISCUSSION TITLE
Changes/addition to costumes

### DISCUSSION THREAD

**Discussion by Amber-Haze** [chromiecraft/chromiecraft#3938] [Suggestions]
As they are, costumes are not fit for roleplaying.
If the goal is a toy that you pop to joke around in the city while waiting in queue, then it works, but usage for RP is incredibly janky, and not worth the price no matter how low it is.

To illustrate what I mean: let's say you want your character to RP as an ogre, the cooldown for the costume is 4 times the duration (10min iirc in this case, so 40min cd), so you'd only be allowed to play In Character for 10 minutes at a time. The work-around would be buying another 3 ogre costumes (different ones, because they share cooldown) and cycle through them which is both annoying to deal with, and breaks character by changing the appearance every 10 minutes (and the fact only a few have 4 versions to make this doable at all).

My suggestion is adding a version without cooldown/duration that costs 4 times the normal one. Hell, make it 5 times if you want. It effectively nets the project the same amount of money for a better quality product (no jank, more immersive, only 1 slot in inventory), or more perhaps, since it'd be a more attractive option for folks like me.
Removing or changing the ones that already exist is not necessary, if someone wants to pop 5k points to have a silly skeleton 5 minutes at a time that's fine, but it's a shame that it's arbitrarily limited as such for no good reason.

I tried them, I got them practically for free because I don't really spend my points on anything and I get them from contributing, and they're gathering dust. Even free, they're not worth it for me.
It's a concept that I really like, a cheap way to play races and stuff that is not allowed by the game otherwise, but the added jank makes them a headache.

Thanks for reading my rant.

**Comment by BeefSteaker** [comment]
Only if we also get the option to disable seeing them too.
